src/components/collapsible.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactNode, FC, useState } from "react"
+import classNames from "classnames"
+
+import ArrowDown from "./icons/arrowDown"
+
+interface Props {
+  /**
+   * False if the element shall be expanded by default
+   * default: true
+   */
+  isInitiallyCollapsed?: boolean
+  /**
+   * A render prop to customize the clickable title of the collapsible element
+   *  - isCollapsed tells you the current state of the collapsible element
+   */
+  renderToggle: (options: { isCollapsed: boolean }) => ReactNode
+  /**
+   * Children to be rendered within the collapse component. The elements will be visible when the component is expanded.
+   */
+  children?: () => ReactNode
+  /**
+   * Function executed when the collapse state has changed.
+   *  - passes the new collapse state as a function parameter
+   */
+  onCollapseStateChange?: (isCollapsed: boolean) => void
+  /**
+   * The size of the arrow down icon
+   * default: 32
+   */
+  collapseIconSize?: number
```

Comment:
Any chance we can go with something like `small` | `large` instead of pixel size?

Reply:
@Averethel  what do you think if I just restrict it to the allowed numbers?

`collapseIconSize?: 24 | 32 | 40`

Reply:
removed it completely after discussion with UX